source/templates.tex

```diff
@@ -6462,7 +6462,8 @@
 
 \pnum
 If a template, a member template or a member of a class template is explicitly
-specialized then that specialization shall be declared before the first use of
+specialized then a declaration of that specialization shall be reachable from
+the first use of
 that specialization that would cause an implicit instantiation to take place,
```

Comment:
Can we simplify this with "every use", as in

> If a template, a member template or a member of a class template is explicitly specialized, a declaration of that specialization shall be reachable from every use of that specialization that would cause an implicit instantiation to take place; no diagnostic is required.

?

Reply:
Fixed and force-pushed.